extensions/slack/package.json

```diff
@@ -18,11 +18,21 @@
   "commands": [
     {
       "name": "search",
-      "title": "Search",
+      "title": "Switch to Channel",
```

Comment:
nit (non-blocking): since this includes users conversations, should we rename it to `Switch to Conversation` or `Open Conversation`?

Reply:
Even Slack has a confusing name for that. On one hand, they call it `Switch to Channel` in the menu bar option:

![CleanShot 2024-06-20 at 10 16 14@2x](https://github.com/raycast/extensions/assets/16003285/b0b2d3a5-d7dc-4be4-ac20-0fe385c77441)

But on the other hand, they call it `Search {Workspace}` in the UI 😄

![CleanShot 2024-06-20 at 10 17 57@2x](https://github.com/raycast/extensions/assets/16003285/f2d2e075-4d4a-4bb2-98bd-342227ad4461)

Aren't user conversations channels as well? This could be named `Open Channel`?

Reply:
I was getting confused in between Users/Groups/Channels as well, and that's why thought `Conversations` could be a good comprehensive name. But `Open Channel` works too.